### ISSUE TITLE
Cannot use the components property on the console command class

### ISSUE THREAD

**Issue by FrancisMawn** [lorisleiva/laravel-actions#205]
Hi,

I was trying to make use of the new **components** property on the Laravel Command class to enhance messages output to the console. So for example, with a regular command class, you can do stuff like 
```php
$this->components->info('Info message');
``` 

However, trying to access the **components** property on the injected Command instance throws the following exception:
```php
Cannot access protected property Lorisleiva\Actions\Decorators\CommandDecorator::$components
```

Am I doing something wrong?

**Comment by leandrodiogenes** [comment]
Hi @FrancisMawn,
try this.
```php
$this->info('Info message');
```

**Comment by leandrodiogenes** [comment]
Im sorry, take some time check the docs in that session.
[https://laravelactions.com/2.x/as-command.html#methods-used](https://laravelactions.com/2.x/as-command.html#methods-used)

```php
namespace App\Actions;

use Illuminate\Console\Command;

class TestCommandAction extends Action {
    public string $commandSignature = 'test-action';

    public function asCommand(Command $command): void {
        $command->info('Done!');
    }

}
```

**Comment by FrancisMawn** [comment]
Im sorry, take some time and read my original post :)

I know I can use the `$command->info()` method. That's what I'm currently using. I was asking for a way to use the **components** instance on the **Illuminate\Console\Command** class.

```php
public function asCommand(Command $command): void 
{
    $command->components->info('Done!');
}

```

**Comment by leandrodiogenes** [comment]
I dont think we can access that property.
As it is a protected property and there is a notice on it.

```php
    /**
     * The console components factory.
     *
     * @var \Illuminate\Console\View\Components\Factory
     *
     * @internal This property is not meant to be used or overwritten outside the framework.
     */
    protected $components;
```

**Comment by leandrodiogenes** [comment]
Im gonna make a pull request to solve that problem, let's hope to be accepted.

**Comment by lorisleiva** [comment]
Thanks for raising this and @leandrodiogenes for the PR!

I've merged and published this as part of `v2.4.1`.
